cmake/Platform/Mac/Install_mac.cmake

```diff
@@ -110,7 +110,7 @@ function(ly_setup_runtime_dependencies_copy_function_override)
 endfunction()
 
 #! ly_install_code_function_override: Mac specific copy function to handle frameworks
-function(ly_install_code_function_override)
```

Comment:
Can you update comment above to reflect new function name as well>

Reply:
sure